packages/suite/src/actions/wallet/send/sendFormSolanaActions.ts

```diff
@@ -0,0 +1,232 @@
+import TrezorConnect, { FeeLevel, TokenInfo } from '@trezor/connect';
```

Comment:
cc @marekrjpolak  this looks like another case for dynamic modules loading in connect we are solving elsewhere. I'd say that these compose logics (eth, sol, cardano) could have methods of their own in connect.

for the purpose of this review I agree to keeping it on this level.

Reply:
ACK

Reply:
yes. however for this PR no action needed yet, we are not ready